### PR TITLE
descriptor_test.go:fix test

### PR DESCRIPTION
**Pull request by q384566678**
Signed-off-by: zhouhao <zhouhao@cn.fujitsu.com>

### CODE REVIEW

**Comment by wking** [comment]
This seems like it should be part of #547.  But I think descriptors
should REQUIRED mediaType [1], in which case we'd need neither.

[1]: https://github.com/opencontainers/image-spec/pull/547#discussion_r99169640

File: schema/content-descriptor.json [hunk 1]
```diff
@@ -26,7 +26,6 @@
     }
   },
   "required": [
-    "mediaType",
```
Comment (stevvooe) [inline schema/content-descriptor.json]: Why are you removing this? `mediaType` is a requirement for descriptors.

**Comment by stevvooe** [comment]
This is also incorrect. Closing.

File: schema/descriptor_test.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ import (
 func TestDescriptor(t *testing.T) {
 	for i, tt := range []struct {
 		descriptor string
-		fail     bool
+		fail       bool
```
Comment (q384566678) [inline schema/descriptor_test.go]: @stevvooe Can you please reopen this PR? I will delete the wrong changes, only to retain this one format changes.thanks.

Comment (stevvooe) [inline schema/descriptor_test.go]: Just open another PR and be clear about what you're actually fixing.

**Pull request closed without merging**